config/config.ts

```diff
@@ -13,5 +13,9 @@
 export const config = {
   server: {
     port: process.env.PORT ? parseInt(process.env.PORT, 10) : 3000
+  },
+  rag: {
+    enableRag: true,
+    vectorStoreDir: "./vectorStore"
```

Comment:
This "default" location for the vector database should be documented.  It's set here to `./vectorStore`, which is a relative path, but it is not clear what "." means in this context.  I had to experiment -- is it the `config` directory itself, or a sub-folder of the `server` directory, or the main `adaptive-palette` directory?  It is the last one, the main `adpative-palette` directory.  It would be helpful if that were documented in `ServerAPS.md`.